.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+<!-- Thank you for sending a PR !-->
```

Comment:
I've seen @ostera's [comment](https://github.com/tldr-pages/tldr/pull/1434#discussion_r128926550), but I don't understand why this line is marked as a comment and the next one isn't. I'd understand if this line was directed at the PR author and the others not (e.g. if the line "Please perform the following checks..." was something similar to "This submission conforms to the project's requirements for PRs", which could be read by both the PR author and other people), but with the current wording, it isn't clear why there's this distinction.

Perhaps this aligns with what @ostera meant about the tone of the message.

My suggestion would be to either comment out both lines and leave only the checks uncommented, or leave the first line commented out and change the wording of the second one to a wording similar to what I suggested above.